src/megacmdshell/megacmdshellcommunications.cpp

```diff
@@ -264,11 +264,11 @@ bool is_pid_running(pid_t pid) {
 }
 #endif
 
-int MegaCmdShellCommunications::createSocket(int number, bool initializeserver, bool net)
+SOCKET MegaCmdShellCommunications::createSocket(int number, bool initializeserver, bool net)
 {
     if (net)
     {
-        int thesock = socket(AF_INET, SOCK_STREAM, 0);
+        SOCKET thesock = socket(AF_INET, SOCK_STREAM, 0);
```

Comment:
int -> SOCKET should also be applied in the else of `if (net)`

Reply:
applied; well spotted